packages/server/src/fhir/operations/export.ts

```diff
@@ -47,27 +47,33 @@ export async function bulkExportHandler(req: Request, res: Response): Promise<vo
 async function exportResourceType(exporter: BulkExporter, project: Project, resourceType: ResourceType): Promise<void> {
   const repo = exporter.repo;
   const hasMore = true;
+  let offset = 0;
   while (hasMore) {
     const bundle = await repo.search({
       resourceType,
       count: 1000,
+      offset,
     });
     if (!bundle.entry || bundle.entry.length === 0) {
       break;
     }
 
-    const writes = [];
     for (const entry of bundle.entry) {
       if (entry.resource?.id) {
-        writes.push(exporter.writeResource(entry.resource));
+        await exporter.writeResource(entry.resource);
```

Comment:
removing using 'Promise.all'. Each time export.writeResource was called `this.writers` was always `{}`. A new writer for each resourceType was always being created https://github.com/medplum/medplum/blob/c03b00e01a25c142811308c31916105a64ae58bd/packages/server/src/fhir/operations/utils/bulkexporter.ts#L54